### PR TITLE
fix(libs): remove line after drop

### PR DESCRIPTION
**Pull request by LuciNyan**
## Description
remove line after drop

Close: #234

### CODE REVIEW

File: libs/components/editor-plugins/src/menu/group-menu/GropuMenu.tsx [hunk 1]
```diff
@@ -115,6 +104,7 @@ export const GroupMenu = function ({ editor, hooks }: GroupMenuProps) {
                 .get(HookType.ON_ROOTNODE_DRAG_END)
                 .subscribe(handleRootDragEnd)
         );
+        sub.add(hooks.get(HookType.ON_ROOTNODE_DROP).subscribe(handleRootDrop));
```
[the remark this answers is not in the document]
Comment (LuciNyan) [reply libs/components/editor-plugins/src/menu/group-menu/GropuMenu.tsx]: Thanks for your detailed response! It has been fixed.

File: libs/components/editor-plugins/src/menu/group-menu/GropuMenu.tsx [hunk 2]
```diff
@@ -170,7 +154,10 @@ export const GroupMenu = function ({ editor, hooks }: GroupMenuProps) {
         setShowMenu(false);
 
         if (groupBlock) {
-            const unobserve = groupBlock.onUpdate(() => setGroupBlock(null));
+            const unobserve = groupBlock.onUpdate(async () => {
+                await sleep();
```
Comment (Austaras) [inline libs/components/editor-plugins/src/menu/group-menu/GropuMenu.tsx:158]: better use `requestAnimationFrame`

Comment (LuciNyan) [reply libs/components/editor-plugins/src/menu/group-menu/GropuMenu.tsx]: Thx

**Review by Austaras — APPROVED** [review]

**Review by SaikaSakura — APPROVED** [review]

**Comment by lawvs** [comment]
Thanks for your pull request, we really appreciate it!

**Pull request merged**